configurator/src/store/modules/keycodes/kb-settings.js

```diff
@@ -230,5 +230,17 @@ export default [
     name: 'RGB Mode G',
     code: 'RGB_M_G',
     title: 'RGB_M_G - RGB Mode: Gradient'
-  }
+  },
+  { name: 'F13', code: 'KC_F13', title: 'KC_F13' },
```

Comment:
let's drop the title key. I'm going to make a change to fall back to code if title isn't specified. This is a change I've been meaning to make. Thanks

Reply:
fixed